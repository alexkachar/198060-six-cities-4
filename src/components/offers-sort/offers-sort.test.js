import React from 'react';
import renderer from 'react-test-renderer';
import {Provider} from 'react-redux';
import {createStore} from 'redux';

import OffersSort from './offers-sort.jsx';

const store = createStore(() => ({
  FILTER: {
    selectedCity: `Amsterdam`,
    selectedFilter: `Popular`
  }
}));

it(`OffersSort renders correctly`, () => {
  const tree = renderer
    .create(
        <Provider store={store}>
          <OffersSort />
        </Provider>, {
          createNodeMock: () => document.createElement(`div`)
        }
    ).toJSON();

  expect(tree).toMatchSnapshot();
});
