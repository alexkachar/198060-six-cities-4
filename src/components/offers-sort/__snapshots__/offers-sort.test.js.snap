// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`OffersSort renders correctly 1`] = `
<form
  action="#"
  className="places__sorting"
  method="get"
>
  <span
    className="places__sorting-caption"
  >
    Sort by 
  </span>
  <span
    className="places__sorting-type"
    onClick={[Function]}
    style={
      Object {
        "outline": "none",
        "userSelect": "none",
      }
    }
    tabIndex={0}
  >
    Popular
    <svg
      className="places__sorting-arrow"
      height={4}
      width={7}
    >
      <use
        xlinkHref="#icon-arrow-select"
      />
    </svg>
  </span>
</form>
`;
